tests/constants.py

```diff
@@ -121,7 +121,7 @@
 MOCK_IP_ADDRESS = '192.0.2.100'
 MOCK_IP_ADDRESS_2 = '203.0.113.20'
 
-MOCK_URSULA_DB_FILEPATH = ':memory:'
+MOCK_URSULA_DB_FILEPATH = tempfile.mkdtemp()
```

Comment:
When does this get cleaned up?

Reply:
In short, the answer is complicated...
It heavily depends on the operating system and the distribution. My all-encompassing-answer guess would be either at boot time or at least once per-day.

In some cases, they're never written to disk and only written to a RAM-disk (if you have a `tmpfs`). I think on OS X, there's some policy about deleting tempfiles if they aren't accessed in a certain amount of time.

Reply:
The docs indicate the following (https://docs.python.org/3.4/library/tempfile.html#tempfile.mkdtemp):

```
The user of mkdtemp() is responsible for deleting the temporary directory and its contents when done with it.
```

If you don't want to handle deletion, an alternative seems to be https://docs.python.org/3.4/library/tempfile.html#tempfile.TemporaryFile